.github/workflows/prepare-release.yml

```diff
@@ -19,8 +19,27 @@ on:
         required: true
         description: Current version (beta, like 0.69.1). Don't include `v`.
 jobs:
+  #validate-version format
+  validate-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Validate version format
+        run: |
+          validate_version() {
+             if [[ ! "$1" =~ ^[1-9][0-9]*\.[1-9][0-9]*\.[1-9][0-9]*(-rcv[0-9]+)?$ ]]; then
```

Comment:
We will need different regexes for beta and stable.  Stable should start with a one or higher and include a `Major.Minor.Patch` version, which no `RC` candidates.  Beta should be the same but the `Major` version can be a zero.